__tests__/app.test.js

```diff
@@ -66,5 +65,40 @@ describe("GET /api/articles/:article_id", () => {
         expect(body.msg).toBe("bad request");
       });
   });
-
+});
+describe("PATCH /api/articles/:article_id", () => {
+  test("200: responds with updated article", () => {
+    const incrementBy = { inc_votes: 5 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(incrementBy)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: expect.any(String),
+          votes: 105,
+        });
+      });
+  });
+  test("status 400: responds with bad request when invalid input", () => {
+    return request(app)
+      .patch("/api/articles/cats")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("bad request");
+      });
+  });
+  test("status 404: not found when passed an invalid endpoint", () => {
+    return request(app)
+      .patch("/api/app")
```

Comment:
Careful, this endpoint should be something like `/api/articles/90000` - almost correct, but trying to find an article that doesn't exist